server/routerlicious/api-report/server-services-client.api.md

```diff
@@ -452,9 +467,17 @@ export interface IWriteSummaryResponse {
 
 // @public (undocumented)
 export class NetworkError extends Error {
-    constructor(
-    code: number, message: string);
+    constructor(// isFatal is enough? -> LETS CREATE OTHER CLASSES
+    code: number, message: string, canRetry?: boolean, isFatal?: boolean, retryAfterMs?: number);
+    // (undocumented)
+    readonly canRetry?: boolean;
     readonly code: number;
+    // (undocumented)
+    get details(): INetworkErrorDetails | string;
+    // (undocumented)
+    readonly isFatal?: boolean;
+    // (undocumented)
+    readonly retryAfterMs?: number;
```

Comment:
Can you document these new properties and methods as you add them?

Reply:
@tylerbutler should I add the comments on server-services-client.api.md or directly in the source file that generated the report?

Reply:
Source file, the api.md is auto-generated I believe

Reply:
Got it. I was just reading about the [API Extractor](https://api-extractor.com/pages/tsdoc/doc_comment_syntax/). Will add the comments there, make sure the API docs get extracted and push a new iteration.

Reply:
I added the comments for TS Doc, and I think the API report is showing that all new properties/methods/classes are now documented. Let me know if there is anything else to fix here regarding documentation. Thanks!